airbyte-db/db-lib/src/test/java/io/airbyte/db/jdbc/TestJdbcUtils.java

```diff
@@ -422,5 +422,10 @@ void testSetStatementSpecialValues(final String colValue, final long value) thro
       assertExpectedOutputTypes(connection);
     }
   }
+  
+  @Test
+  void testParseJdbcParameters() {
+    Map<String, String> parameters = JdbcUtils.parseJdbcParameters("sessionVariables=max_execution_time=10000", "$");
```

Comment:
Can you assert the test with the expected result?
Can you add more complex input, maybe use the example you tried in the PR image.